opennlp-tools-models/src/test/java/opennlp/tools/models/AbstractModelLoaderTest.java

```diff
@@ -33,6 +33,8 @@ public void testLoadOpenNLPModel() throws Exception {
         model.getModelSHA256());
     assertEquals("langdetect-183.bin", model.getModelName());
     assertEquals("1.8.3", model.getModelVersion());
+    assertEquals("unknown"
+        /* "root" <- correct value after OpenNLP models 1.0.1 */, model.getModelLanguage());
```

Comment:
This line will get changed after releasing the 1.0.1 models?

Reply:
Yep. We can also wait until 1.0.1 is ready.

Reply:
> Yep. We can also wait until 1.0.1 is ready.

Marked PR as draft.

Reply:
Updated PR. Ready now.